Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.yml

```diff
@@ -0,0 +1,2098 @@
+category: Data Enrichment & Threat Intelligence
```

Comment:
Same question about the category here. 
@meital-aizen

Reply:
@ShacharKidor Correct, it should be changed to Network Security. FYI @RamboV